File: C/pattern4.c
// #This is simple reverse-pyramid pattern c-program 
#include<stdio.h>
int main(){

    for (int i =0;i<=5;i++){
         for (int j =0;j<=i;j++){
            printf(" ");
         }
         for (int h =5;h>i;h--){
             printf("* ");
         }
        printf("\n");
    }

    return 0;
}

File: Python/pattern4.py
#This is simple reverse-pyramid pattern python-program 
for i in range(0,5,1):
    for j in range(0,i,1):
        print(" ",end="")
    for k in range(4,i,-1):
        print("*",end=" ")
    print("*",end="\n")
